### ISSUE TITLE
I have a problem

### ISSUE THREAD

**Issue by txj-mssl** [Schweini07/Dialogbox#2]
Hello, I trying this plug-in. Show_text adds multiple texts to the text array, but sometimes it doesn't display the second, sometimes it displays the second.

The second text in the array cannot be displayed correctly when running the scenario
Sometimes, it can display correctly,

**Comment by txj-mssl** [comment]
I used Chinese fonts on dialog nodes

**Comment by Schweini07** [comment]
This could be because you have the show_text twice in a row in your code. But could you explain a bit further? When it's not working try downloading and implementing the master branch into your project.

**Comment by txj-mssl** [comment]
[youwenti.zip](https://github.com/Schweini07/Dialogbox/files/4425816/youwenti.zip)
This is a show project

**Comment by txj-mssl** [comment]
发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

发件人: Schweini07<mailto:notifications@github.com>
发送时间: 2020年4月3日 14:40
收件人: Schweini07/Dialogbox<mailto:Dialogbox@noreply.github.com>
抄送: txj-mssl<mailto:txjjzyzqbx@hotmail.com>; Author<mailto:author@noreply.github.com>
主题: Re: [Schweini07/Dialogbox] I have a problem (#2)


This could be because you have the show_text twice in a row in your code. But could you explain a bit further? When it's not working try downloading and implementing the master branch into your project.

―

**Comment by Schweini07** [comment]
Hmm.. It works for me everytime

**Comment by txj-mssl** [comment]
![image](https://user-images.githubusercontent.com/58028249/78339760-95c6d900-75c7-11ea-97f6-15cb3bc07438.png)

**Comment by txj-mssl** [comment]
I don't know if you can get the video

**Comment by Schweini07** [comment]
What do you mean with that?

**Comment by txj-mssl** [comment]
I just cleared my email and sent you a video showing the problems, but I replied directly through the address sent by hithub. I wonder if you can receive the video，Sorry, my English is terrible

**Comment by Schweini07** [comment]
You can't send it to the github address. Try to upload it somewhere and then send a link

**Comment by txj-mssl** [comment]
https://kdocs.cn/l/sn1NYcqMb?f=141
[文件] 2020-04-03 16-18-21.mkv

**Comment by txj-mssl** [comment]
Click the blue button in the middle to download. Can you open the web page, please

**Comment by Schweini07** [comment]
Doing it

**Comment by Schweini07** [comment]
Ok, that's interesting. But I can't reproduce it somehow, can you maybe explain a bit more what you do when this happens?

**Comment by txj-mssl** [comment]
I just click the left mouse button when the first text is finished playing, or wait for a few seconds when the first text is finished playing, and then click again. This happens. I just did it later

**Comment by Schweini07** [comment]
I tried hard but can't reproduce it, you maybe could try to turn `use speedup` off. Maybe it works then?

**Comment by txj-mssl** [comment]
I close use speedup，without effect

**Comment by Schweini07** [comment]
Ok then try this: Connect the input_triggered signal from the dialogbox to the control node. When it's called it should print("triggered"). Tell me if this happens once or twice, when the error occures

**Comment by txj-mssl** [comment]
![image](https://user-images.githubusercontent.com/58028249/78342902-9d3cb100-75cc-11ea-8db4-9470bd6a215c.png)

**Comment by txj-mssl** [comment]
Normal or abnormal, printing once

**Comment by Schweini07** [comment]
Try it with dialog_finished

**Comment by txj-mssl** [comment]
I'm going to cook dinner and I'll reply to you later

**Comment by Schweini07** [comment]
What you could also try is to open the showcase.tscn file in the resources folder and look if it works there

**Comment by txj-mssl** [comment]
![image](https://user-images.githubusercontent.com/58028249/78353053-05df5a00-75dc-11ea-902e-e95d22c01974.png)
Printed but not shown

**Comment by txj-mssl** [comment]
No display, sometimes display
![image](https://user-images.githubusercontent.com/58028249/78353437-9ddd4380-75dc-11ea-9975-b1e1bb1ba204.png)

**Comment by Schweini07** [comment]
An assumption is that this was caused trough the work with character expressions, I've updated it, try downloading the master branch again

**Comment by txj-mssl** [comment]
I downloaded it again and used it in the project. This problem still exists

**Comment by Schweini07** [comment]
I am really confused, I'll take a break now and look at it back later

**Comment by txj-mssl** [comment]
question progress？

**Comment by Schweini07** [comment]
I probably finish 0.2 today, maybe I'll find and fix the bug

**Comment by txj-mssl** [comment]
thanks

**Comment by Schweini07** [comment]
I tried it on a windows laptop and I still couldn't reproduce the bug. 

Could you try to use an other trigger than the mouse or/and use multiple dialogs?

**Comment by txj-mssl** [comment]
Just now, I downloaded the main branch again. I still have problems using UI accept

**Comment by txj-mssl** [comment]
I try to use multiple dialog boxes, using the same trigger, but the problem occurs at the same time
![image](https://user-images.githubusercontent.com/58028249/78520549-90e96b80-77f9-11ea-828d-6a2ec9bb954f.png)

**Comment by Schweini07** [comment]
I meant that: $Dialogbox.show_text(["AAAAAAAAAAAAA", "BBBBBBBBBBBBB", "CCCCCCCCCCCCC"])

**Comment by txj-mssl** [comment]
![image](https://user-images.githubusercontent.com/58028249/78546779-97470a00-7830-11ea-94d1-a0b3fd39d515.png)
bug

**Comment by txj-mssl** [comment]
I've tried

**Comment by Schweini07** [comment]
I tried it on multiple devices and it works all the time with me. 
Have you tried it on another device?

**Comment by txj-mssl** [comment]
no，my Only one computer

**Comment by Schweini07** [comment]
If you have the opportunity to try it on another please use it. I assume this is a hardware problem and related to Godot. I sadly have to publish 0.2 without a potential fix

**Comment by txj-mssl** [comment]
I please a friend for a test. He said there was no problem

**Comment by txj-mssl** [comment]
He said  didn't get a bug

**Comment by Schweini07** [comment]
Yes, then it is probably hardware related. I hope this doesn't stop you from using the plugin
